examples/tree_3d_dgsem/elixir_mhd_dynamo.jl

```diff
@@ -0,0 +1,165 @@
+# For a comparison look into the Pencil Code example https://github.com/pencil-code/pencil-code/tree/master/samples/helical-MHDturb.
+
+using OrdinaryDiffEq
+using Trixi
+using Random
+using LinearAlgebra
+using StableRNGs
+
+###############################################################################
+# initial condition and forcing function
+
+"""
+    initial_condition_gaussian_noise_mhd(x, t, equations)
+
+Define the initial condition with a weak small-scale Gaussian noise magnetic field.
+"""
+function initial_condition_gaussian_noise_mhd(x, t, equations)
+    amplitude = 1e-8
+
+    seed = reinterpret(UInt, t + x[1] + x[2] + x[3])
+    rng = StableRNG(seed)
+
+    rho = 1.0
+    rho_v1 = 0.0
+    rho_v2 = 0.0
+    rho_v3 = 0.0
+    rho_e = 10.0
+    B1 = randn(rng) * amplitude
+    B2 = randn(rng) * amplitude
+    B3 = randn(rng) * amplitude
+    psi = 0.0
+
+    return SVector(rho, rho_v1, rho_v2, rho_v3, rho_e, B1, B2, B3, psi)
+end
+
+"""
+    source_terms_helical_forcing(u, x, t, equations::CompressibleMhdDiffusion3D)
+
+Forcing term that adds a helical small-scale driver to the system that is
+delta-correlated in time.
+"""
+function source_terms_helical_forcing(u, x, t, equations::IdealGlmMhdEquations3D)
+    # forcing amplitude
+    f_0 = 0.02
+    # helicality of the forcing (-1, 1)
+    sigma = 1.0
+
+    # Extract some parameters for the computation.
+    c_s = 0.1
+    delta_t = 0.01
+
+    # To make sure that the random numbers depend only on time we need to set the seeds.
+    seed = reinterpret(UInt, t)
+    rng = StableRNG(seed)
+
+    # Random phase -pi < phi <= pi
+    phi = (randn(rng) * 2 - 1) * pi
+
+    # Random vector k, also delta-correlated in time.
+    # k = SVector((randn(rng) * 2 - 1) * pi, (randn(rng) * 2 - 1) * pi, (randn(rng) * 2 - 1) * pi)
+    k = [0.0, 0.0, 0.0]
+    k[1] = (randn(rng) * 2 - 1) * pi
+    k[2] = (randn(rng) * 2 - 1) * pi
+    k[3] = (randn(rng) * 2 - 1) * pi
+    k = k / norm(k)
+    k = 6 * k
+    k_norm = k / norm(k)
+
+    # Random unit vector, not aligned with k.
+    ee = SVector(randn(rng), randn(rng), randn(rng)) .* 2 .- 1
+    # Normalize the vector e.
+    ee = ee / norm(ee)
+
+    # Compute the f-vectors.
+    f_nohel = cross(k, ee) / sqrt(dot(k, k) - dot(k, ee)^2)
+    M = [0 k_norm[3] -k_norm[2]; -k_norm[3] 0 k_norm[1]; k_norm[2] -k_norm[1] 0]
+    R = (I - im * sigma * M) / sqrt(1 + sigma^2)
+    f_k = R * f_nohel
+
+    # Normalization factor to make sure that the time integration of f is 0.
+    N = f_0 * c_s * sqrt(norm(k) * c_s / delta_t)
+
+    forcing = real(N * f_k * exp(im * dot(k, x) + im * phi)) / u[1]
+
+    return SVector(0, forcing[1], forcing[2], forcing[3], 0, 0, 0, 0, 0)
+end
+
+
+###############################################################################
+# semidiscretization of the resistive compressible MHD equations
+
+prandtl_number() = 0.72
+# Corresponds to previous alpha^2 runs.
+mu() = 2e-3
+eta = 2e-3
```

Comment:
The PENCIL code link given at the beginning of this file recommends these quantities to be `5e-3` why pick different values?